static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -65,6 +67,7 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
             };
           }}
           onClick={() => {
+            markAsRead({readUpdate: true, showSuccessToast: false});
```

Comment:
This is not the only way to see a feedback. It's also possible to load up a feedback via url/link to the whole page.

We should move this to a different spot. For example at the top of feedbackItem.tsx inside a `useEffect` callback.